Casks/a/amazon-workspaces.rb

```diff
@@ -9,11 +9,11 @@
   homepage "https://clients.amazonworkspaces.com/"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url "https://docs.aws.amazon.com/workspaces/latest/userguide/amazon-workspaces-osx-client.html"
+    regex(/>\s*v?(\d+(?:\.\d+)+)[^<]*</i)
```

Comment:
```suggestion
    regex(/>\s*v?(\d+(?:\.\d+)+)\s*</i)
```

Using `[^<]*` may lead to unwanted matches, so there's something to be said for simply using `\s*` after the version (as we do in other similar regexes).

For example, this regex also matches "3.0" from the "3.0+ client" text in a separate table. This doesn't cause a problem at the moment because 3.0 is lower than the newest version (5.21.0) and there was also a 3.0.0 release but the looseness could become a problem in the future.